arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -461,7 +542,7 @@
         "tags": [
           "Queues"
         ],
-        "operationId": "Queues_ListByNamspace",
+        "operationId": "Queues_ListByNamspaces",
```

Comment:
This should be `Queues_ListByNamspace`? otherwise the generated code will looks like:

```java
class Queues {
  List<Queue> ListByNamespaces(String namespaceName);
}
```

method name gave a wrong hint that it can take multiple namespace names but only one can be provided.